builtin/credential/cert/path_config.go

```diff
@@ -13,9 +13,10 @@ func pathConfig(b *backend) *framework.Path {
 		Pattern: "config",
 		Fields: map[string]*framework.FieldSchema{
 			"disable_binding": &framework.FieldSchema{
-				Type:        framework.TypeBool,
-				Default:     false,
-				Description: `If set, during renewal, skips the matching of presented client identity with the client identity used during login. Defaults to false.`,
+				Type:         framework.TypeBool,
+				Default:      false,
+				Description:  `If set, during renewal, skips the matching of presented client identity with the client identity used during login. Defaults to false.`,
+				DisplayValue: false,
```

Comment:
Can we make DisplayValue default to Default when not provided?  I would much rather have that to limit the duplication.

Reply:
@kalafut as of now the `Default` field doesn't make it into the path help response at all. Is this something we can change?

Reply:
Should be able to, yes. The original discussion of this was complicated because _adding_ defaults to the Go code just for display purposes has some undesirable side effects, but I think for those where it's present, there's no issue emitting them.

Reply:
This has been implemented and merged into 1.1 beta.